packages/features/calendar-cache/lib/datesForCache.ts

```diff
@@ -9,7 +9,7 @@ export const getTimeMin = (timeMin?: string) => {
 /** Expand the end date to the end of the next month */
 export function getTimeMax(timeMax?: string) {
   const date = timeMax ? new Date(timeMax) : new Date();
-  date.setMonth(date.getMonth() + 2);
+  date.setMonth(date.getMonth() + 1);
```

Comment:
Reduced the range

Reply:
Why are we reducing?

Reply:
Google tops at 90 days. If we stumble with a  31 + 30  + 31 month combo it will fail. It was 30 before #18224 so it will still be around 60 now.